spec/laa_crime_schemas/structs/return_details_spec.rb

```diff
@@ -47,5 +47,21 @@
         expect { subject }.to raise_error(Dry::Struct::Error, /invalid type for :returned_at/)
       end
     end
+
+    context 'with split case reason' do
+      let(:valid_details) do
+        JSON.parse(
+          LaaCrimeSchemas.fixture(1.0, name: 'application_returned_split_case').read
+        )['return_details']
+      end
+
+      let(:attributes) { valid_details }
+
+      it 'builds the return details struct' do
+        expect(subject.reason).to eq('split_case')
+        expect(subject.details).to eq('Offense 1 reason requires more detail')
```

Comment:
I think we use the Offence spelling elsewhere as it is more common in the UK - Offense is the american spelling I think.